website/docs/cloud-docs/stacks/deploy/plans.mdx

```diff
@@ -29,9 +29,12 @@ A stack’s **Overview** page displays the following information:
 
 * The number of your stack’s components and deployments  
 * Your deployment’s current [health](#check-deployment-health)  
-* The latest configuration version  
+* The latest configuration version
+* The inputs and outputs for your stack   
 * A chart listing each deployment’s recent configuration versions
 
+Clicking on a Stack’s **Inputs** reveals a list of the inputs this Stacks is using, along with the upstream Stacks that supply those inputs. Clicking on a Stack’s **Outputs** reveals a list of the outputs of this Stack, and any downstream Stacks that intake those outputs. To learn more about passing data from one Stack to another, refer to [Pass data from one Stack to another](/terraform/language/stacks/deploy/link-stacks).
```

Comment:
the stack inputs page currently represents the inputs sourced from an upstream stack, it does not list out inputs from other sources. In the case where all the inputs are sourced from upstream stacks then the list represents all the stack's input. If some inputs are sourced from somewhere else, then the input list will be a partial list of the stack's input.

Reply:
Gotcha, I see now I was too vague in this description - thank you for catching that!!

 @Uk1288  Would saying something like:
```suggestion
Clicking on a Stack’s **Inputs** reveals a list of the inputs this Stack has access to from `upstream_input` blocks, along with the upstream Stacks that supply those inputs. Clicking on a Stack’s **Outputs** reveals a list of the outputs of this Stack and any downstream Stacks that intake those outputs. To learn more about passing data from one Stack to another, refer to [Pass data from one Stack to another](/terraform/language/stacks/deploy/link-stacks).
```

Make more sense to you? Or be clearer?

Reply:
yes, exactly right!